site/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "build": "bundle exec jekyll build",
+    "dev": "bundle exec jekyll serve",
```

Comment:
I don't mind a single command but I imagine it could be confusing if someone were to make some changes and not see them appear because the cached version of the file is being used (totally done this before plenty of times 🤕).

But then again it's good to have your dev env as close to prod as possible and 'Bypass for network' is a good way around the cache whilst still using the service worker.